week-1/K-functions-parameters/exercise2.js

```diff
@@ -1,5 +1,7 @@
-// Declare your function first
-
+// Declare your function first/
+function divide(num1, num2) {
+  return num1 / num2;
+}
 var result = divide(3, 4);
 
```

Comment:
great. Well done too on naming the parameters in a way that is consistent with the function's responsibility.